File: apps/web/src/app/router.ts
import { createRouter, createWebHistory } from 'vue-router'

export const router = createRouter({
  history: createWebHistory(),
  routes: [
    { path: '/', name: 'dashboard', component: () => import('../features/dashboard/DashboardView.vue') },
    { path: '/market', name: 'market', component: () => import('../features/market/MarketView.vue') },
    { path: '/strategy', name: 'strategy', component: () => import('../features/strategy/StrategyView.vue') },
    { path: '/backtest', name: 'backtest', component: () => import('../features/backtest/BacktestView.vue') },
    { path: '/ai-analysis', name: 'ai-analysis', component: () => import('../features/ai-analysis/AiAnalysisView.vue') },
    { path: '/trading', name: 'trading', component: () => import('../features/trading/TradingView.vue') },
    { path: '/risk', name: 'risk', component: () => import('../features/risk/RiskView.vue') },
    { path: '/logs', name: 'logs', component: () => import('../features/logs/LogsView.vue') },
    { path: '/settings', name: 'settings', component: () => import('../features/settings/SettingsView.vue') },
  ],
})

File: apps/api/app/api/v1/router.py
from fastapi import APIRouter

from app.api.v1.routes import (
    ai_analysis,
    backtests,
    dashboard,
    health,
    logs,
    market,
    risk,
    settings,
    strategies,
    system,
    trading,
)

api_router = APIRouter()
api_router.include_router(health.router, tags=["health"])
api_router.include_router(system.router, prefix="/system", tags=["system"])
api_router.include_router(dashboard.router, prefix="/dashboard", tags=["dashboard"])
api_router.include_router(market.router, prefix="/market", tags=["market"])
api_router.include_router(strategies.router, prefix="/strategies", tags=["strategies"])
api_router.include_router(backtests.router, prefix="/backtests", tags=["backtests"])
api_router.include_router(ai_analysis.router, prefix="/ai-analysis", tags=["ai-analysis"])
api_router.include_router(trading.router, prefix="/trading", tags=["trading"])
api_router.include_router(risk.router, prefix="/risk", tags=["risk"])
api_router.include_router(logs.router, prefix="/logs", tags=["logs"])
api_router.include_router(settings.router, prefix="/settings", tags=["settings"])

